erpnext/projects/doctype/time_log/time_log.json

```diff
@@ -12,49 +12,51 @@
  "fields": [
   {
    "allow_on_submit": 0, 
-   "bold": 1, 
```

Comment:
yes, cause standard changed some fields order, I made changes to make Time Log form look similar to before.